CHANGELOG.unreleased.md

```diff
@@ -11,6 +11,7 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- [Perf Alert] Stop raising an alert for the `XCP-ng Tools` SR storage usage (PR [#6755](https://github.com/vatesfr/xen-orchestra/pull/6755))
```

Comment:
You need to update the message to reflect the PR changes (user oriented message).

Also update the title of the PR (commit message) to reflect the change (dev oriented message).